src/ui/components/pages/otherAccounts.module.css

```diff
@@ -10,10 +10,17 @@
   z-index: 2;
   top: 0;
   padding: 14px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
   border-bottom: 1px solid #edf0f4;
   background: #fff;
 }
 
+.header .addAccountButton {
```

Comment:
Better extract the rule defining `margin` and use it where needed than overriding existing rule when it's not needed